### ISSUE TITLE
Launch with button icons not appearing on dev.dockstore.net

### ISSUE THREAD

**Issue by coverbeck** [dockstore/dockstore#3380]
**Describe the bug**
The icons for Terra, AnVIL and DNAnexus are not present in the Launch with panel.

**To Reproduce**
Go to https://dev.dockstore.net/workflows/github.com/DataBiosphere/topmed-workflows/UM_variant_caller_wdl:feature/checker-optional-crai?tab=info

**Expected behavior**
Should see the icons

**Screenshots**
![Screen Shot 2020-04-07 at 4 41 13 PM](https://user-images.githubusercontent.com/1049340/78729375-b6a46c80-78ee-11ea-9c2b-61aea8f72278.png)

**Additional context**

Pretty sure this was caused by the [commit to add custom headers](https://github.com/dockstore/dockstore-ui2/commit/d208b149e9a62dcda57c144a74eb37daf793d78e). The custom headers are getting sent to CF, which doesn't recognize them as allowed headers for a CORS request.

The interceptor should only add the headers if it is a request to the same domain as what the UI is on.

┆Issue is synchronized with this [Jira Story](https://ucsc-cgl.atlassian.net/browse/DOCK-1301)
┆Issue Type: Story
┆Fix Versions: Dockstore 1.9
┆Sprint: Sprint 33 Herring
┆Issue Number: DOCK-1301

**Comment by denis-yuen** [comment]
> The interceptor should only add the headers if it is a request to the same domain as what the UI is on.

On the other hand, should it? I could imagine scenarios where we might want to see who/what is hitting CF. Can we add the new headers as allowed headers for a CORS request?

**Comment by coverbeck** [comment]
> On the other hand, should it? I could imagine scenarios where we might want to see who/what is hitting CF. Can we add the new headers as allowed headers for a CORS request?

Hmm, interesting, when I wrote that I was thinking generically, i.e., what if one day the UI calls some thirdparty service. And then I remembered that we already do, cwlviewer. So I checked that just now, and it does work. Looks like they allow all headers.

So I could see allowing the new headers for our CF, but to be safe, I think we should remove the headers from non-Dockstore domains.
